### ISSUE TITLE
Status of each k in verbose mode

### ISSUE THREAD

**Issue by matthewlehew** [nikita-moor/ldatuning#22]
It would be a great enhancement if verbose mode could indicate when it had finished fitting the model for each k in the `topics` parameter, even something as simple as `Model fit for k=50`. Doing so would provide a de facto progress meter, allowing the user to see how quickly the `FindTopicsNumber()` job is progressing instead of having no idea when it will complete. 

It's okay if the k values appear out of order due to parallel processing; what's important is seeing how quickly the models are being fit.

**Comment by titaniumtroop** [comment]
Agreed that it would be desirable to have some sort of progress meter. The parallel step in this package is a single call to an anonymous function. Perhaps the anonymous function in the call to `parallel::parLapply` could `cat` the value of `k` (which is `x` in the call to the anonymous function) after the call to the `topicmodels::LDA` function is complete.

I have tried to get a reliable progress meter working with parallel processing in other projects, but I wasn't ever happy with the results because they don't seem to get displayed on the screen in a timely fashion.

I'm the maintainer of the package, but I have not been actively developing the package in several months. If you have a pull request that passes CI, I'd be happy to take a look.
